packages/expo-ui/src/jetpack-compose/TextInput/index.tsx

```diff
@@ -61,6 +61,19 @@ export type TextInputProps = {
    * @default true
    */
   autocorrection?: boolean;
+  /**
+   * Options to request software keyboard to capitalize the text. Applies to languages which has upper-case and lower-case letters.
+   *
+   * Available options:
+   * - characters
+   * - none
+   * - sentences
+   * - unspecified
+   * - words
```

Comment:
Let's add the descriptions from the [android docs](https://developer.android.com/reference/kotlin/androidx/compose/ui/text/input/KeyboardCapitalization#Unspecified()) for more clarity 
```suggestion
   * - `characters`:  Capitalize all characters.
   * - `none`: Do not auto-capitalize text.
   * - `sentences`: Capitalize the first character of each sentence.
   * - `unspecified`: Capitalization behavior is not specified.
   * - `words`: Capitalize the first character of every word.
```